less/modules/site.less

```diff
@@ -56,6 +56,15 @@ html {
 			background-color: @darkGray;
 		}
 	}
+	.control-panel-picker__center {
+		position: absolute;
+		left: 50%;
+		height: 27px;
+		line-height:27px;
+		padding: 0px 20px;
```

Comment:
FE standards say this should be... 
`padding: 0 20px;`

Although I'd be surprised if it's taken into consideration anywhere else :p